### PR TITLE
function plainAddPlaceholder remove metadata of pdf (#36)

### PR DESCRIPTION
**Pull request by cdebevotte**
Fix for: function plainAddPlaceholder remove metadata of pdf (#36)

### CODE REVIEW

File: src/helpers/plainAddPlaceholder/createBufferTrailer.js [hunk 1]
```diff
@@ -15,9 +15,9 @@ const createBufferTrailer = (pdf, info, addedReferences) => {
         Buffer.from('\ntrailer\n'),
         Buffer.from('<<\n'),
         Buffer.from(`/Size ${info.xref.maxIndex + 1}\n`),
-        Buffer.from(`/Prev ${info.xRefPosition}\n`),
         Buffer.from(`/Root ${info.rootRef}\n`),
-        Buffer.from('/Info 15 0 R\n'),
```
Comment (vbuch) [inline src/helpers/plainAddPlaceholder/createBufferTrailer.js:20]: Wow! That's a nice hardcoded bit :-)

**Review by vbuch — CHANGES_REQUESTED** [review]
Just make sure coverage remains at 100% and this gets merged.
Thank you!

**Comment by cdebevotte** [comment]
This is my first time doing a PR, I'm not very used to it :-)

**Review by vbuch — APPROVED** [review]

**Comment by vbuch** [comment]
It's in develop. Will try to get it released these days. Thank you for the contribution @cdebevotte

**Pull request merged**